webui/src/pages/repositories/repository/settings/branches.jsx

```diff
@@ -47,8 +47,8 @@ const SettingsContainer = () => {
                     <Card className={"w-100 rounded border-0"}>
                         <Card.Body className={"p-0 rounded"}>
                             <ListGroup>
-                                {rules && rules.length > 0 ? rules.map(r => {
-                                    return <ListGroup.Item >
+                                {rules && rules.length > 0 ? rules.map((r, idx) => {
+                                    return <ListGroup.Item key={idx}>
```

Comment:
IIRC array indexes make for bad keys in React lists, because they may be reused with different values.

Reply:
Very true. Fixed.